### PR TITLE
Si ff module tasks

### PR DESCRIPTION
**Pull request by boonier**

### CODE REVIEW

File: blocks/core/ff_module/ff_module-task/_ff_module-task-renderer.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+var Task = require('./ff_module-task.js'),
+    selector = 'data-ff_module-task';
+
+var props = {
+    modifier: 'wibble',
+    classes: 'another-class',
+    from: 'Big Daddy',
+    to: 'A Class',
+    duedate: "27/08/2018",
+    message: 'A big message from Big Daddy',
```
Comment (roobottom) [inline blocks/core/ff_module/ff_module-task/_ff_module-task-renderer.js]: This made me laugh, but our pattern lib is publicly accessible. Can we change this to more meaningful test data?

File: blocks/core/ff_module/ff_module-task/ff_module-task.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<notice id="{{ id }}" display_date_type="{{ display_date_type }}" sent_to="{{ sent_to }}" no_excused="{{ no_excused }}" handedin="{{ completed_by }}" marked="{{ marked }}" type="onlinetask" from="{{ from }}" from_href="{{ from_href }}" link_href="{{ link_href }}" to="{{ to }}" duedate="{{ duedate }}" startdate="{{ startdate }}" setdate="{{ setdate }}">
+<notice modifier="{{ modifier }}" id="{{ id }}" sent_to="{{ sent_to }}" no_excused="{{ no_excused }}" handedin="{{ completed_by }}" marked="{{ marked }}" from="{{ from }}" link_href="{{ link_href }}" to="{{ to }}" duedate="{{ duedate }}">
```
Comment (roobottom) [inline blocks/core/ff_module/ff_module-task/ff_module-task.xml:1]: Did you remove some attrs here for a reason?

**Comment by roobottom** [comment]
@boonier @dkhuntrods see inline comments. Looks mostly all good.

File: blocks/core/ff_module/ff_module-task/ff_module-task.xml (hunk 2 above)
Comment (boonier) [inline blocks/core/ff_module/ff_module-task/ff_module-task.xml:1]: They weren't being used anywhere in the xsl, so made what was being used less clear.

File: blocks/core/ff_module/ff_module-task/_ff_module-task-renderer.js (hunk 1 above)
Comment (boonier) [inline blocks/core/ff_module/ff_module-task/_ff_module-task-renderer.js]: Have updated now.

**Pull request merged**